utils/makeWebpackConfig.js

```diff
@@ -88,6 +88,10 @@ function getModule() {
         test: /\.js$/,
         exclude: /node_modules/,
         loader: 'babel-loader'
+      },
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader']
```

Comment:
style-loader gives us the ability to require css files in our js files. css-loader then interprets the loaded css file. i read that works like a chain of unix shell commands.